src/config/serde/ethereum_address.rs

```diff
@@ -0,0 +1,67 @@
+use crate::swap::ethereum;
+use serde::{de, export::fmt, Deserialize, Deserializer, Serializer};
+use std::str::FromStr;
+
+pub fn deserialize<'de, D>(deserializer: D) -> Result<Option<ethereum::Address>, D::Error>
```

Comment:
❓ Why do we need a custom deserialiser for this now? Isn't the one from the `comit` library sufficient?

Reply:
It is surprisingly not. Just take my test change and try it locally ;)
Also, our test to use `toml::from_str` instead of using `config_rs` like it is done in the `main` was useless.